### DISCUSSION TITLE
Welcome to go-workflows Discussions!

### DISCUSSION THREAD

**Discussion by cschleiden** [cschleiden/go-workflows#36] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Robinmxc** [comment]
MySQL 5.7 does not support the SKIP LOCKED function. It is only supported in versions above MySQL 8.0. Do you want to consider supporting lower versions of MySQL？

**Reply by cschleiden** [reply]
With 5.7 end-of-life approaching soon I'd like to focus on the newer version(s) for now.
